packages/directory/addon/templates/directory.hbs

```diff
@@ -22,6 +22,7 @@
   {{dir-sidebar classNames='directory__sidebar'}}
 
   <section class='directory__table'>
+    <h1>My Data</h1>
```

Comment:
The title won't always be "My Data" when you're in the directory. We need to make this title depend on the child route being accessed.

Reply:
fixed